rundeckapp/grails-app/services/rundeck/services/JobMetadataService.groovy

```diff
@@ -88,7 +88,7 @@ class JobMetadataService {
      * @param project project
      */
     def removeProjectPluginMeta(final String project, final String type) {
-        pluginMetaDataProvider.deleteByProjectAndDataKey(project, type)
+        pluginMetaDataProvider.deleteAllByProjectAndKeyLike(project, type)
```

Comment:
original `deleteByProjectAndDataKey` converted parameter `type` into a LIKE query using `%/${type}`, so that should be what is passed in here.